games-simulation/singularity/singularity-1.00.recipe

```diff
@@ -20,22 +20,22 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	cmd:python3$secondaryArchSuffix
 	numpy
```

Comment:
Have you tried to run this with only the listed packages?

Reply:
Yes, same errors.

Reply:
I mean, it needs numpy_python3, not numpy

Reply:
It could be changed to that, but at the moment I do not think that I can test it reliably since both my VM and netbook have been unable to update from the main Haiku package repo. The HaikuPorts pacakge repo remains fine here, though. Would that be acceptable still?

Reply:
Using `numpy_python3` results in this terminal output:

```
Traceback (most recent call last):
  File "./Singularity", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/packages/python3-3.7.9-1/.self/lib/python3.7/vendor-packages/pkg_resources/__init__.py", line 3251, in <module>
    @_call_aside
  File "/packages/python3-3.7.9-1/.self/lib/python3.7/vendor-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/packages/python3-3.7.9-1/.self/lib/python3.7/vendor-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/packages/python3-3.7.9-1/.self/lib/python3.7/vendor-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/packages/python3-3.7.9-1/.self/lib/python3.7/vendor-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/packages/python3-3.7.9-1/.self/lib/python3.7/vendor-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'polib>=0.7' distribution was not found and is required by Endgame-Singularity
```
Unfortunately, I couldn't find `polib` within the HaikuPorts repo.

Reply:
Installed `polib` via `pip3`, then ran the game from the terminal. This is the output:
https://gist.github.com/win8linux/550cdb89b501e2c10750a5025b4ecc5d

Highly doubt that packaging or porting `polib` would make much of a difference at all, as it crashes in almost the exact same way without `polib` installed at all and using `numpy` instead of `numpy_python3`.